runner/webserver.ts

```diff
@@ -63,6 +63,15 @@ export function webserver(wct: Context): void {
     const wsOptions = options.webserver;
     const additionalRoutes = new Map<string, RequestHandler>();
 
+    const pathToLocalWct =
+        path.join(options.root, 'bower_components', 'web-component-tester');
+    if (!exists(pathToLocalWct)) {
+      throw new Error(`WCT isn't installed locally. Run:
+    bower install --save-dev web-component-tester
```

Comment:
I think we should expand this message. Maybe something like:

```
The web-component-tester Bower package is not installed as a dependency of this project.

Please run this command to install:
    bower install --save-dev web-component-tester

Web Component Tester 6.0 now requires that support files needed by the browser are installed as part of the project's dependencies or dev-dependencies. This is so that projects are ensured to get the correct version of support files to match their version of web-component-tester.
```